src/components/transactions/TxDetails/TxData/SpendingLimits/styles.module.css

```diff
@@ -9,9 +9,3 @@
   flex-direction: column;
   gap: 4px;
 }
-
-.inline {
```

Comment:
This class is still being used in one place. Do we need it?

Reply:
Nice catch! I've restored it:
![image](https://user-images.githubusercontent.com/20442784/186375541-ec5a6432-733d-41ff-90d6-d8ab2b9f251f.png)